app/controllers/links_controller.rb

```diff
@@ -1,20 +1,15 @@
 require 'local-links-manager/export/link_status_exporter'
 
 class LinksController < ApplicationController
-  before_action :load_dependencies
+  before_action :load_dependencies, :set_back_url_before_post_request
```

Comment:
is this relevant to all controller methods, or could it be restricted to just one or two?